lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -909,7 +909,7 @@ def sanitize_network_params(self):
                                           " a VLAN name under VM network list.")
 
             if 'name' in network and find_obj(self.content, [vim.Network], network['name']) is None:
-                self.module.fail_json(msg="Network '%(name)s' does not exists" % network)
+                self.module.fail_json(msg="Network '%(name)s' does not exist." % network)
```

Comment:
The integrations tests need to be changed as well.

Reply:
Can you tell me where to find these?
Searching for "does not exists" didn't give any results.
However I found more spelling mistakes that I can fix.

Reply:
https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest/tasks/network_negative_test.yml#L68

Reply:
https://github.com/ansible/ansible/blob/f8b85bdbf3b1ee4ac1a80cd6da852d1a1ab1a2fe/test/integration/targets/vmware_guest/tasks/network_negative_test.yml#L68

Reply:
@mkrizek , done.